encrypted-media/scripts/invalid-license.js

```diff
@@ -0,0 +1,31 @@
+function runTest(config) {
+    async_test(function (test) {
+        var initDataType;
+        var initData;
+        var keySystem = config.keysystem;
+        var invalidLicense = new Uint8Array(
+          [0x00, 0x11, 0x22, 0x33, 0x44, 0x55, 0x66, 0x77]);
+
+        function handleMessage(event) {
+            event.target.update(invalidLicense).then(function (event) {
+                assert_unreached('Error: update() succeeded unexpectedly.');
+                test.done();
```

Comment:
You don't need to call test.done() in case of the test failure. After assert_unreached the test will fail and test.done() won't be executed.